spec/views/job_alert_feedbacks/edit.html.haml_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'job_alert_feedbacks/edit.html.haml' do
+  before do
+    assign(:feedback_form, JobAlertFeedbackForm.new)
+    allow(view).to receive(:subscription_feedback_path).and_return('/stubbed-path')
+  end
+
+  context 'recaptcha' do
```

Comment:
I'm not sure about the usefulness of view specs like this

Reply:
Each one takes 0.5 seconds on my computer. Maybe you're right. I'll get rid of these.